README.md

```diff
@@ -172,7 +172,7 @@ You can also specify negated paths in the `exclude` array, by prefixing them wit
 Negated paths can restore paths that have been already been excluded in the `exclude` array.
 Multiple `exclude` globs can be specified on the command line, each must follow a `--exclude`, `-x` switch.
 
-The default `exclude` list is defined in the [test-exclude module](https://github.com/istanbuljs/istanbuljs/blob/master/packages/test-exclude/default-exclude.js).
+The default `exclude` list is defined in the [test-exclude module](https://github.com/istanbuljs/test-exclude/blob/master/default-exclude.js).
```

Comment:
Good catch on the move, though `test-exclude` will soon be altered to pull the default from our shared schema module so I'd prefer pointing to the final location.

```suggestion
The default `exclude` list is defined in the [@istanbuljs/schema module](https://github.com/istanbuljs/schema/blob/master/default-exclude.js).
```